### PR TITLE
Prepopulated db support for all dists

### PR DESCRIPTION
**Pull request by benjaoming**
Supersedes #4743

### CODE REVIEW

File: kalite/distributed/management/commands/setup.py [hunk 1]
```diff
@@ -287,30 +286,41 @@ def handle(self, *args, **options):
...
-               or not raw_input_yn("Remove database file '%s' now? " % database_file) \
-               or not raw_input_yn("WARNING: all data will be lost!  Are you sure? "):
-                install_clean = False
-                print("Upgrading database to KA Lite version %s" % VERSION)
-            else:
+        database_exists = database_file and os.path.isfile(database_file)
+
+        # An empty file is created automatically even when the database dosn't
+        # exist. But if it's empty, it's safe to overwrite.
+        database_exists = database_exists and os.path.getsize(database_file) > 0
```
Comment (MCGallaspy) [inline kalite/distributed/management/commands/setup.py]: Make it a one-liner with 294?

**Comment by MCGallaspy** [comment]
Looks ok.

File: kalite/distributed/management/commands/setup.py (hunk 1 above)
Comment (benjaoming) [inline kalite/distributed/management/commands/setup.py]: For the sake of the comment :)

File: kalite/distributed/management/commands/setup.py [hunk 2]
```diff
@@ -287,30 +286,41 @@ def handle(self, *args, **options):
...
-            print("WARNING: Database file already exists!")
-            print(
-                "-------------------------------------------------------------------")
-            if not options["interactive"] \
-               or raw_input_yn("Keep database file and upgrade to KA Lite version %s? " % VERSION) \
-               or not raw_input_yn("Remove database file '%s' now? " % database_file) \
-               or not raw_input_yn("WARNING: all data will be lost!  Are you sure? "):
-                install_clean = False
-                print("Upgrading database to KA Lite version %s" % VERSION)
-            else:
```
Comment (benjaoming) [inline kalite/distributed/management/commands/setup.py]: All this wasnt deleted but moved down

**Comment by MCGallaspy** [comment]
Hmm, I tried this out by creating a distributable using `make dist`, then installing (`ka-lite-static`). I get the following error:

```text
(kalite-venv-test) C:\Users\Mike\.kalite2>kalite start --foreground
Running 'kalite start' in foreground...

Stand by while the server loads its data...

INFO:kalite:Setting up KA Lite; this may take a few minutes; please wait!


   _   __  ___    _     _ _
  | | / / / _ \  | |   (_) |
  | |/ / / /_\ \ | |    _| |_ ___
  |    \ |  _  | | |   | | __/ _ \
  | |\  \| | | | | |___| | ||  __/
  \_| \_/\_| |_/ \_____/_|\__\___|

https://learningequality.org/ka-lite/

         version 0.16.0

Cleaning *pyc files (if writable) from: C:\Users\Mike\Envs\kalite-venv-test\shar
e\kalite\dist-packages
Cleaning *pyc files (if writable) from: C:\Users\Mike\Envs\kalite-venv-test\shar
e\kalite\python-packages
(Re)moving database file to temp location, starting clean install. Recovery loca
tion: c:\users\mike\appdata\local\temp\tmpasjr9k
Traceback (most recent call last):
  File "C:\Users\Mike\Envs\kalite-venv-test\share\kalite\python-packages\django\
core\management\base.py", line 224, in run_from_argv
    self.execute(*args, **options.__dict__)
  File "C:\Users\Mike\Envs\kalite-venv-test\share\kalite\python-packages\django\
core\management\base.py", line 263, in execute
    output = self.handle(*args, **options)
  File "C:\Users\Mike\Envs\kalite-venv-test\lib\site-packages\kalite\distributed
\management\commands\initialize_kalite.py", line 41, in handle
    self.setup_server_if_needed()
  File "C:\Users\Mike\Envs\kalite-venv-test\lib\site-packages\kalite\distributed
\management\commands\initialize_kalite.py", line 34, in setup_server_if_needed
    call_command("setup", interactive=False)
  File "C:\Users\Mike\Envs\kalite-venv-test\share\kalite\python-packages\django\
core\management\__init__.py", line 161, in call_command
    return klass.execute(*args, **defaults)
  File "C:\Users\Mike\Envs\kalite-venv-test\share\kalite\python-packages\django\
core\management\base.py", line 263, in execute
    output = self.handle(*args, **options)
  File "C:\Users\Mike\Envs\kalite-venv-test\lib\site-packages\kalite\distributed
\management\commands\setup.py", line 384, in handle
    shutil.move(database_file, dest_file)
  File "C:\Python27\Lib\shutil.py", line 303, in move
    os.unlink(src)
WindowsError: [Error 32] The process cannot access the file because it is being
used by another process: 'C:\\Users\\Mike\\.kalite2\\database\\data.sqlite'
```

**Comment by MCGallaspy** [comment]
Solution is to close the db connection prior to calling setup mgmt command:
```python
from django import db
db.close_connection()
...
```

Patch incoming.

**Pull request merged**